tests/CSharpHandlerTest.php

```diff
@@ -2,11 +2,25 @@
 
 namespace Redberry\Spear\Tests;
 
-use Redberry\Spear\Spear;
-use PHPUnit\Framework\TestCase;
+use Redberry\Spear\Facades\Spear;
+use Orchestra\Testbench\TestCase;
 
 class CSharpHandlerTest extends TestCase
 {
+	/**
+	 * Get package providers.
+	 *
+	 * @param \Illuminate\Foundation\Application $app
+	 *
+	 * @return array
+	 */
+	protected function getPackageProviders($app)
+	{
+		return [
+			'Redberry\Spear\ServiceProvider',
+		];
+	}
+
```

Comment:
Tazo it will be better if you'll do one base test case and then set the getPackageProviders method and then extend that TestCase class